### ISSUE TITLE
rhea ETL does not add labels for directional forms

### ISSUE THREAD

**Issue by cmungall** [biopragmatics/obo-db-ingest#9]
https://raw.githubusercontent.com/biopragmatics/obo-db-ingest/main/export/rhea/122/rhea.obo

only the direction-neutral (`=`) forms have a name/label

**Comment by cthoyt** [comment]
Good point. Rhea doesn't assign names to the non-top-level reaction. Do you think it would be good to duplicate the name? Or add some extra tags like `<base reaction> (left to right)`?

**Comment by cmungall** [comment]
should we move issues like this to pyobo?

I think it should be sufficient to simply change the operator symbol (=, <=, =>, <=>). There is an argument for suffixing all labels to be extra-clear but here we start to diverge a bit from an isomorphic database mapping to something more opinionated. Not necessarily wrong but it should be done intentionally and transparently
